### PR TITLE
Improve proteomics tutorials, fix links

### PR DESCRIPTION
**Pull request by Stortebecker**
commented at https://gitter.im/Galaxy-Training-Network/Lobby

### CODE REVIEW

**Comment by Stortebecker** [comment]
Huh? Why can't I see the [travis results](https://travis-ci.org/galaxyproject/training-material/builds/202150829) any more?

And I do not comprehend why travis failed. The link is correct. Is there some general problem with links to the PRIDE archive? Also the first link was kinda correct, just linking the wrong content...

**Comment by martenson** [comment]
@Stortebecker because travis won't run on a conflicting branch

File: Proteomics/tutorials/proteinID_SG_PS.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Identifying the proteins contained in a sample is an important step in any prote
...
 
-As an example dataset, we will use an LC-MS/MS analysis of HeLa cell lysate published in [Vaudel et al., 2014, Proteomics](https://www.ncbi.nlm.nih.gov/pubmed/24678044). Detailed information about the dataset can be found on [PRIDE](https://www.ebi.ac.uk/pride/archive/projects/PXD001907) . For step 2 we will use a validated human Uniprot FASTA database without appended decoys. If you already completed the tutorial on [Database Handling](./database-handling.md) you can use the constructed database before the **DecoyDatabase** :wrench: step.
+As an example dataset, we will use an LC-MS/MS analysis of HeLa cell lysate published in [Vaudel et al., 2014, Proteomics](https://www.ncbi.nlm.nih.gov/pubmed/24678044). Detailed information about the dataset can be found on [PRIDE](https://www.ebi.ac.uk/pride/archive/projects/PXD000674) . For step 2 we will use a validated human Uniprot FASTA database without appended decoys. If you already completed the tutorial on [Database Handling](./database-handling.md) you can use the constructed database before the **DecoyDatabase** :wrench: step.
```
Comment (bgruening) [inline Proteomics/tutorials/proteinID_SG_PS.md]: @Stortebecker as a general remark, if you use more line breaks such diffs are more readable.

File: Proteomics/tutorials/WIP_proteinQuant_SIL.md [hunk 2]
```diff
@@ -7,18 +7,32 @@ tutorial_name: proteinQuant_SIL
...
 
-This tutorial deals with protein quantitation via stable isotope labelling (SIL). We will use tools of the OpenMS suite. The tutorial covers relative quantitation only (i.e. comparison of abundances in different samples, no absolute quantitation of peptides / proteins).
+This tutorial deals with protein quantitation via stable isotope labelling (SIL). We will use tools of the OpenMS suite. The tutorial covers relative quantitation only (i.e. comparison of abundances in different samples, no absolute quantitation of peptides / proteins). We tutorial covers quantitation only, i.e. you need to perform peptide and protein ID in beforehand. to learn about protein ID in Galaxy, please consider [this tutorial](./proteinID_SG_PS.md).
```
Comment (bgruening) [inline Proteomics/tutorials/WIP_proteinQuant_SIL.md]: `We tutorial`? 
`. to learn` ...

**Comment by bgruening** [comment]
I just solved the merge conflicts, you should be ready to go @Stortebecker!

**Comment by martenson** [comment]
Thanks @Stortebecker !

**Pull request merged**